### PR TITLE
Add quantity queries

### PR DESCRIPTION
**Pull request by jordanmaslyn**
I love what you did and wanted to help round it out by adding quantity queries from http://quantityqueries.com/.

### CODE REVIEW

**Comment by LukyVj** [comment]
Hey man, sorry for the time I take to merge this one /o\ ( busy week ) 
I'll merge that tomorow! Thanks a lot for adding those man ✨

File: source/stylesheets/vendors/_family.scss [hunk 1]
```diff
@@ -187,3 +187,33 @@
   }
   @return $i
 }
+
+// Quantity Queries - http://quantityqueries.com/
+//////////////////
```
Comment (LukyVj) [inline source/stylesheets/vendors/_family.scss:192]: I think I'll get rid of this. 
I'd like to keep the source file as light as possible. 

I'll mention the quantityqueries.com in the readme. How does it sounds for you ?

**Comment by jordanmaslyn** [comment]
Sounds good to me. I just wanted to make sure to attribute the source since I didn't come up with those brilliant queries myself. I just made them mixins.

**Comment by LukyVj** [comment]
Hehe no problem. That's what I did for family.scss tho. Taking some complicated css apis, and turned them into mixins.

**Comment by LukyVj** [comment]
Fixed in 003472a51b7db3e25068c00c63121e78cca96bf2

**Pull request closed without merging**